pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdNamespaces.java

```diff
@@ -1808,12 +1801,13 @@ void run() throws PulsarAdminException {
                    description = "Maximum number of bytes in a topic backlog before compaction is triggered "
                                  + "(eg: 10M, 16G, 3T). 0 disables automatic compaction",
                    required = true)
-        private String threshold = "0";
+        private String thresholdStr = "0";
```

Comment:
These "xxxxStr" modifications seem not necessary?  IMHO, the previous way is cleaner.

Reply:
I think the previous code is not easy to read, and it’s not a good way to pass the method in the method, I prefer to pass a variable.

Reply:
OK, it is only a matter of taste.